kubernetes_state/metadata.csv

```diff
@@ -5,7 +5,7 @@ kubernetes_state.node.pods_capacity,gauge,,,,The total pod resources of the node
 kubernetes_state.node.cpu_allocatable,gauge,,cpu,,The CPU resources of a node that are available for scheduling,0,kubernetes,k8s_state.node.cpu_allocatable
 kubernetes_state.node.memory_allocatable,gauge,,byte,,The memory resources of a node that are available for scheduling,0,kubernetes,k8s_state.node.memory_allocatable
 kubernetes_state.node.pods_allocatable,gauge,,,,The pod resources of a node that are available for scheduling,0,kubernetes,k8s_state.node.pods_allocatable
-kubernetes_state.node.status,gauge,,,,The number of nodes in a given status, per tag,0,kubernetes,k8s_state.node.status
+kubernetes_state.node.status,gauge,,,,The number of nodes by schedulability; tagged 'status:schedulable' and 'status:unschedulable'.,0,kubernetes,k8s_state.node.status
```

Comment:
Actually, re-reading [the check](https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/check.py#L253-L264), I'm not sure my new description is right, either. "Number of nodes" doesn't factor into it; the metric value always has a value of 1.

I don't understand this metric.